src/screens/Dashboard/TransactionList.tsx

```diff
@@ -107,6 +111,7 @@ const styles = StyleSheet.create({
   noTransactionsContainer: {
     alignItems: 'center',
   },
+  footer: { height: 500 },
```

Comment:
This looks cheesy

Reply:
It moves a scroll view to the top when you got only one transaction after search. Should it be done in a different way?